src/renderer/helpers/fp/eq.test.ts

```diff
@@ -153,6 +154,20 @@ describe('helpers/fp/eq', () => {
     })
   })
 
+  describe('eqONullableString', () => {
+    it('is equal', () => {
+      expect(eqONullableString.equals(O.some('MEMO'), O.some('MEMO'))).toBeTruthy()
```

Comment:
Just to complete all cases:
`expect(eqONullableString.equals(undefined, undefined)).toBeTruthy()`